auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -461,21 +577,274 @@ export default defineComponent({
       }
     }
 
-    watch(() => props.selectedColumns, (newCol: string[]) => {
-      getHeaders(newCol)
-    })
+    const enableNewActionsMenu = (): boolean => {
+      return launchdarklyServices.getFlag(LDFlags.EnableNewActionsMenu) || false
+    }
+
+    const isModernizedEntity = (item: Business): boolean => {
+      const entityType = getEntityType(item)
+      const supportedEntityFlags = launchdarklyServices.getFlag(LDFlags.IaSupportedEntities)?.split(' ') || [] // move to somewhere factory?
+      console.log(supportedEntityFlags)
+      return supportedEntityFlags.includes(entityType)
+    }
+
+    const isColinEntity = (item: Business): boolean => { // not include ones modernized (feature flag)
+      const entityType = getEntityType(item)
+      // talk to Sev to see if there's better way to check colin
+      return (!isModernizedEntity(item) && (
+        entityType === CorpTypes.BC_COMPANY ||
+        entityType === CorpTypes.BC_CCC ||
+        entityType === CorpTypes.BC_ULC_COMPANY ||
+        entityType === CorpTypes.CCC_CONTINUE_IN ||
+        entityType === CorpTypes.BC_CORPORATION ||
+        entityType === CorpTypes.ULC_CONTINUE_IN ||
+        entityType === CorpTypes.EXTRA_PRO_REG ||
+        entityType === CorpTypes.LL_PARTNERSHIP ||
+        entityType === CorpTypes.LIMITED_CO ||
+        entityType === CorpTypes.LIM_PARTNERSHIP ||
+        entityType === CorpTypes.BC_UNLIMITED ||
+        entityType === CorpTypes.XPRO_LL_PARTNR ||
+        entityType === CorpTypes.XPRO_LIM_PARTNR
+      ))
+    }
+
+    const isSocieties = (item: Business): boolean => {
+      const entityType = getEntityType(item)
+      return (entityType === CorpTypes.CONT_IN_SOCIETY ||
+        entityType === CorpTypes.SOCIETY ||
+        entityType === CorpTypes.XPRO_SOCIETY)
+    }
+
+    const isOtherEntities = (item: Business): boolean => {
+      const entityType = getEntityType(item)
+      return (entityType === CorpTypes.FINANCIAL ||
+        entityType === CorpTypes.PRIVATE_ACT ||
+        entityType === CorpTypes.PARISHES)
+    }
+
+    const isAllowRestore = (item: Business): boolean => {
```

Comment:
This should check a FF also.

There is a FF for this in LD "entity-ui" project but THIS app uses a different LD project. I really don't want to duplicate this flag in "sbc-auth" project. @seeker25 Any ideas?

![image](https://github.com/bcgov/sbc-auth/assets/10002889/d5f4cee7-8d88-4f8b-9fdb-0c1d0993e600)

Reply:
We could add a label for auth-web - It looks like it's possible to have two client instances.  Would probably make double the usage though?

Reply:
The label isn't used externally, I think.

Two client instances would be nice, but yeah, it might count twice towards usage.

I wonder if we should move to a single LaunchDarkly project for sbc-auth + entity-ui?

Reply:
I created a new FF (in LD sbc-auth project), `supported-restoration-entities`, that you can use here to determine whether the subject entity type can be restored in LEAR or not.

In Dev and Test, this flag should return "BC BEN CC ULC", and in Prod, "".

Reply:
Yeah, so this is a duplicate of the flag in entity-ui, but it's a temporary flag so hopefully we remember to remove both in the future when restorations is fully implemented.

Reply:
Now using FF (in #2433 ). Resolving.